.graphqlconfig

```diff
@@ -2,7 +2,7 @@
   "name": "Subsquid GraphQL Schema",
   "schemaPath": [
     "https://origin.squids.live/origin-squid/graphql",
-    "https://squid.subsquid.io/prime-eth-squid/graphql",
+    "https://origin.squids.live/prime-eth-squid/v/v13/graphql",
```

Comment:
I assume you'll go back to a prod URL on this before merging?

Reply:
oh, nice catch